torchbenchmark/models/BERT_pytorch/__init__.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, test, device, batch_size=None, jit=False, extra_args=[]):
             '--vocab_path', f'{root}/data/vocab.small',
             '--output_path', 'bert.model',
         ]) # Avoid reading sys.argv here
-        args.with_cuda = self.device == 'cuda'
+        args.with_cuda = self.device == 'cuda' or self.device == 'mps'
```

Comment:
Piggybacking on the cuda flag as i didn't want to create additional changes in the function signature.

Reply:
I suggest we remove the `with_cuda` flag and replace it with `device` and pass in `self.device`.